test/user.spec.js

```diff
@@ -382,4 +388,313 @@ describe('POST /user', () => {
+
+        id = user._id;
+
+        done();
+    })
+    .catch(err => {
+        done(new Error(err));
+    });
+
+  });
+
+  it('Should update', (done) => {
+
+    let update_user = {
+        email: 'juancho_great@live.com.mx',
+        name: 'Juan Ernesto',
+        phone: '2223454590',
+        language: 'Inglés',
+        country: 'Chile'
+    };
+
+    chai.request(server)
+    .put('/users/'+id)
+    .send(update_user)
+    .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.a('object');
+        res.body.should.have.property('phone');
+        res.body.should.have.property('country');
+        res.body.should.have.property('language');
+        res.body.should.have.property('email');
+        res.body.should.have.property('name');
+        done();
+    });
+  });
+
+  it('Wrong id', (done) => {
+
+    let id = '5db48a252f3af03923defe7f';
+
+    let update_user = {
+        email: 'juancho_great@live.com.mx',
+        name: 'Juan Ernesto',
+        phone: '2223454590',
+        language: 'Inglés',
+        country: 'Chile'
+    };
+
+    chai.request(server)
+    .put('/users/' + id)
+    .send(update_user)
+    .end((err, res) => {
+      shouldBeNotFound(res, done);
+    });
+
+  });
+
+  it('User name too short', (done) => {
+
+    let update_user = {
+      email: 'juancho_great@live.com.mx',
+      name: 'U',
+      phone: '2223454590',
+      language: 'Inglés',
+      country: 'Chile'
+    };
+    chai.request(server)
+      .put('/users/'+id)
+      .send(update_user)
+      .end((err, res) => {
+        shouldBeError(res, done, Errors.SHORT_STRING);
+      });
+  });
+
+  it('User name too long', (done) => {
+
+    let update_user = {
+      email: 'juancho_great@live.com.mx',
+      name: 'abcdefghifabcdefghifabcdefghifabcdefghifabcdefghifabcdefghifabcdefghifabcdefghifabcdefghifabcdefghifabcdefghifabcdefghifangjolnb',
+      phone: '2223454590',
+      language: 'Inglés',
+      country: 'Chile'
+    };
+    chai.request(server)
+      .put('/users/'+id)
+      .send(update_user)
+      .end((err, res) => {
+        shouldBeError(res, done, Errors.LONG_STRING);
+      });
+  });
+
+  it('Non alphabetic chars in name', (done) => {
+
+    let update_user = {
+      email: 'juancho_great@live.com.mx',
+      name: 'Juan Ernesto 1',
+      phone: '2223454590',
+      language: 'Inglés',
+      country: 'Chile'
+    };
+    chai.request(server)
+      .put('/users/'+id)
+      .send(update_user)
+      .end((err, res) => {
+        shouldBeError(res, done, Errors.INVALID_FORMAT);
+      });
+  });
+
+  it('Phone too short', (done) => {
+
+    let update_user = {
+      email: 'juancho_great@live.com.mx',
+      name: 'Juan Ernesto',
+      phone: '222345',
+      language: 'Inglés',
+      country: 'Chile'
+    };
+    chai.request(server)
+      .put('/users/'+id)
+      .send(update_user)
+      .end((err, res) => {
+        shouldBeError(res, done, Errors.NUMBER_LOWER_BOUND);

```

Comment:
Phone should be treated as string, not number.

Reply:
But its property in the model is a Number

Reply:
> Phone should be treated as string, not number.

But its property in the model is a Number